### PR TITLE
Lastpresentation

### PR DESCRIPTION
**Pull request by fprados**

### CODE REVIEW

File: _posts/2020-01-20-workshop.md [hunk 1]
```diff
@@ -22,16 +22,16 @@ clinicians from around the world and discuss together and establish new collabor
...
+| 11:30 - 13:30 | <b>Session 4</b><br>Olivier Vincent (Polytechnique Montreal - Canada)<br><i>Generalizing deep learning segmentation across multiple MRI contrasts and centers</i><br><br>Junqian Gordon Xu (Baylor College of Medicine - Houston - USA)<br><i>A pilot two-center reproducibility study of quantitative cervical spinal cord MRI for clinical trials in progressive multiple sclerosis</i><br><br>Govind Bhagavatheeshwaran (National Institute of Neurological Disorder and Stroke - USA)<br><i>Assessing the role of spinal cord atrophy in neuroinflammatory diseases</i><br><br>Frederik Barkhof (Centre for Medical Image Computing - UCL - UK)<br><i>Spinal cord atrophy rates in randomised controled trials</i><br><br> |
```
Comment (jcohenadad) [inline _posts/2020-01-20-workshop.md]: He usually goes by the name "Govind Nair"

Comment (fprados) [reply _posts/2020-01-20-workshop.md]: ok

**Comment by fprados** [comment]
Could you approve this and then I will change the name?

**Comment by jcohenadad** [comment]
> Could you approve this and then I will change the name?

Why can’t you change the name in a new commit in this branch?

**Comment by fprados** [comment]
now

**Pull request merged**